### PR TITLE
Use urlopen from urllib.request to download models instead of wget

### PR DESCRIPTION
**Pull request by jcwchen**
**Description**
- Use urlopen (from Python standard library: urllib) to download a file instead of wget: add a new function _download_file
- Remove wget dependency
- Move master branch to main branch for onnx/models (The migration in onnx/models was just done today)

**Motivation and Context**
wget was brought by ONNX 1.11 new feature: `onnx.hub`. However, wget is not included in standard Python library and it has not been updated for quite a while (Last version was published 6 years ago). 

Another proposal for using requests:
```
def _download_file(url: str, filename: str) -> None:
    """
    Downloads the file with specifed filename from the url
    @param url: a url of download link
    @param filename: a specified file name for the downloaded file
    """
    with requests.get(url, stream=True) as response:
        response.raise_for_status()
        with open(filename, 'wb') as f:
            for chunk in response.iter_content(chunk_size=8192):
                f.write(chunk)
```
I finally chose to simply use urlopen from urllib because it's from Python standard library so ONNX doesn't need to install additional library.

### CODE REVIEW

**Review by liqunfu — APPROVED** [review]

File: onnx/test/hub_test.py [hunk 1]
```diff
@@ -9,14 +9,10 @@
 import os
 
 
-@pytest.mark.skipif(
-    'TEST_HUB' not in os.environ or not os.environ['TEST_HUB'],
-    reason="Conserving Git LFS quota"
-)
```
[the remark this answers is not in the document]
Comment (jcwchen) [reply onnx/test/hub_test.py:15]: It passes all CIs and I am adding it back.

File: onnx/hub.py [hunk 2]
```diff
@@ -122,22 +121,38 @@ def _get_base_url(repo: str, lfs: bool = False) -> str:
         return "https://raw.githubusercontent.com/{}/{}/{}/".format(repo_owner, repo_name, repo_ref)
 
 
+def _download_file(url: str, file_name: str) -> None:
+    """
+    Downloads the file with specifed file_name from the url
+    @param url: a url of download link
```
Comment (xadupre) [inline onnx/hub.py:127]: `:param`

**Review by xadupre — APPROVED** [review]

File: onnx/hub.py (hunk 2 above)
Comment (jcwchen) [reply onnx/hub.py]: Good point. `:param` might be more prevalent than `@param`. Have just updated it globally in the same file. Thanks!

**Review by mhamilton723 — APPROVED** [review]

**Comment by mhamilton723** [comment]
Thank you @jcwchen!

**Pull request merged**